src/pydex_app/orderbook.py

```diff
@@ -3,17 +3,28 @@
 
 author: officialcryptomaster@gmail.com
 """
-
+from pydex_app.database import PYDEX_DB as db
 from pydex_app.db_models import SignedOrder
+from pydex_app.order_watcher_client import OrderWatcherClient as owc
 from pydex_app.utils import paginate
 
 
 class Orderbook:
     """Abstraction for orderbook of signed orders
     """
+    _owc = owc()
+
     def __init__(self):
         pass
 
+    @classmethod
+    def add_order(cls, order):
```

Comment:
I think we need a separate service to keep at least part of orderbook in memory... it can get very expensive very quick to keep everything from disk. Other than than, the main problem with what you have here is the order_watcher client will be instantiated every time there is a request and killed when that request ends... We need to run order watcher clients separately and communicate with them... I sugest you remove the order watcher stuff from this code for now while I figure it out...